ui-v2/app/routes/dc/acls/index.js

```diff
@@ -7,14 +7,26 @@ import WithAclActions from 'consul-ui/mixins/acl/with-actions';
 
 export default Route.extend(WithAclActions, {
   repo: service('repository/acl'),
+  settings: service('settings'),
   queryParams: {
     s: {
       as: 'filter',
       replace: true,
     },
   },
   beforeModel: function(transition) {
-    return this.replaceWith('dc.acls.tokens');
+    return get(this, 'settings')
+      .findBySlug('token')
+      .then(token => {
+        // If you don't have a token set or you have a
+        // token set with AccessorID set to null (legacy mode)
+        // then rewrite to the new acls
+        if (!token || get(token, 'AccessorID') !== null) {
```

Comment:
Shouldn't that be `get(token, 'AccessorID') === null`? Right now the code doesn't match the comment.

Reply:
Just read the code below, looks like it's the comment that's wrong, not the code.

Reply:
Ah! hehe yeah sorted!